views/layouts/default.html.erb

```diff
@@ -29,13 +29,14 @@
             <nav class="pd-b">
               <a href="/" title="Welcome">Welcome</a>
               <a href="/#sec-hardfork" title="Bigger&nbsp;blocks">Bigger&nbsp;blocks</a>
+              <a href="/onboard.html" title="Who's Onboard">Who's&nbsp;Onboard</a>
```

Comment:
@NxtChg You may want to resize your browser window to test how this longer menu works with the responsive layout. For instance, it doesn't display well at 1000px wide (it doesn't fit space and overflows under the logo misaligned) and 800px wide (the welcome link ends up over the orange background and isn't super readable).